pkg/ccl/streamingccl/streamingest/stream_ingestion_processor.go

```diff
@@ -448,33 +448,38 @@ func (sip *streamIngestionProcessor) Start(ctx context.Context) {
 			return
 		}
 		subscriptions[id] = sub
-		sip.subscriptionGroup.GoCtx(sub.Subscribe)
+		sip.subscriptionGroup.GoCtx(func(ctx context.Context) error {
+			if err := sub.Subscribe(ctx); err != nil {
+				sip.sendError(errors.Wrap(err, "subscription finished with error"))
+			}
+			return nil
+		})
 	}
 
 	sip.mergedSubscription = mergeSubscriptions(sip.Ctx(), subscriptions)
 	sip.workerGroup.GoCtx(func(ctx context.Context) error {
 		if err := sip.mergedSubscription.Run(); err != nil {
-			sip.sendError(err)
+			sip.sendError(errors.Wrap(err, "merge subscription finished with error"))
```

Comment:
[nit] Let's be consistent in our wrapping. ("with error" vs "failed") My preference would be to just go with "merged subscription" since "failed" and "error" are implicit and 9 times of of 10 the error message itself also has a lot of verbiage.

Reply:
Done.